draft-fossati-tls-attestation.md

```diff
@@ -815,7 +815,7 @@ including anti-money laundering, drug development in healthcare, contact
 tracing in pandemic times, etc.
 
 In such scenarios, the users (e.g., the party providing the data input for the
-computation, the consumer of the computed attestation results, the party
+computation, the party consuming the computed results, or the party
```

Comment:
I think, we need to work more on the example here! Suddenly we introduced the term `party providing a proprietary ML model` without setting the context.

Reply:
Agree with @yogeshbdeshpande. There is no mention of ML model before this point in time. A little more context should be provided here. 
Additionally, the abbreviation ML is not spelled out.

Reply:
Sure, the examples can always be improved.  The scope of this PR is narrower though: it's only meant to fix a bug in the actors' description.

Reply:
> Additionally, the abbreviation ML is not spelled out.

fixed in 459df76cf06ea8a17d140300ccf405d33df9dd90

Reply:
> I think, we need to work more on the example here! Suddenly we introduced the term `party providing a proprietary ML model` without setting the context.

fixed in d82eceed8438f3b44cde0170330a049134964243